strawberryfields/gbs/points.py

```diff
@@ -66,3 +68,34 @@ def kernel(R: np.ndarray, sigma: float) -> np.ndarray:
         K (array): the kernel matrix
     """
     return np.exp(-(scipy.spatial.distance.cdist(R, R)) ** 2 / 2 / sigma ** 2)
+
+
+def perpp(R: np.ndarray, sigma: float, n_mean: int, ns: int) -> np.ndarray:
```

Comment:
What is the motivation for having `R` and `sigma` as inputs as opposed to the kernel matrix directly?
It necessarily constrains us to using the RBF kernel (which is actually maybe something we want to do at this point).

Reply:
The main reason is that the user does not have to create the kernel and then sample from it. The user just calls the wrapper function. But I think it is better to separate them: the user calls both and the sampler gets the kernel matrix as input.